### ISSUE TITLE
Design Intent - Accessibility Feedback - Cartographers (MHV on VA.gov) for Landing Page Onsite Notifications

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#63787]
## VFS actions
- [ ] Assign this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] Comment on this ticket:
  - [ ] If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - [ ] When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - [ ] When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] Close the ticket when all feedback has been addressed.

## Thoughts/questions

-

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- **Must:** No strong color preferences, except to [verify any color choices](https://webaim.org/resources/contrastchecker/) against the background and adjacent elements. WCAG minimum contrast for an element like this is 3:1, but for something like this there's no reason you couldn't do better than 7:1.
- **Must:** I don't have a strong feeling on whether the dot indicator should be on the left or the right, but it definitely needs to be in relatively close proximity to the link text. Too far apart and screen magnification users may miss the indicator entirely.
- **Must:** Now is a great time to think about the screen reader text that will go along with the dot --- what you want to say, what order it goes in with the link, how the link text and the screen reader text sound together grammatically. 
   - **Consider:** In terms of accessibility, I'll always be a fan of visible text on the page. If you've got screen reader text that you like, consider whether there's a way to display that text for everyone! Definitely not a must, but something to explore.
- **Should:** Now is also a great time to start thinking about how the dot indicator behaves when the link is in different states, especially `:focus` and `:hover`. Does it change color to match with the link? Does it get bigger or change shape? etc.
